scripts/enforce-call-super-for-overrided-onupdate

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+function is_call_super_found {
+  # If class directly extends BaseProjectKeyAvailabilityAssuranceAction then call super.onUpdate is no needed
+  grep "extends BaseProjectKeyAvailabilityAssuranceAction" "$0" > /dev/null
+  if [[ $? -eq 0 ]]
+  then
+    return 0
+  fi
+
+  # If no call super.onUpdate found then return 255 that cause xargs failure
+  grep "super.onUpdate" "$0" > /dev/null
+  if [[ $? -ne 0 ]]
+  then
+    echo Call 'super.onUpdate' not found in "$0"
+    return 255
+  fi
+}
+
+export -f is_call_super_found
+
+git grep --files-with-matches "void onUpdate" | grep --invert-match "BaseProjectKeyAvailabilityAssuranceAction.java" | xargs -n 1 bash -c "is_call_super_found $@" 2> /dev/null
+
+if [[ $? -ne 0 ]]
+then
+  echo "You must call 'super.onUpdate' if you override 'onUpdate' method and this class is not directly extends 'BaseProjectKeyAvailabilityAssuranceAction'!"
```

Comment:
This doesn't fail with exit code != 0 and is thus easy to overlook, pls use `die` from utils.sh as in other scripts

Reply:
`s/is not directly extends/does not directly extend`